extensions/resource-deployment/src/ui/notebookWizard/notebookWizardPage.ts

```diff
@@ -7,43 +7,68 @@ import { EOL } from 'os';
 import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
 import { NotebookWizardPageInfo } from '../../interfaces';
-import { initializeWizardPage, InputComponents, InputComponent, setModelValues, Validator } from '../modelViewUtils';
+import { getTextComponent, initializeWizardPage, InputComponent, InputComponents, setModelValues, Validator } from '../modelViewUtils';
 import { WizardPageBase } from '../wizardPageBase';
 import { NotebookWizard } from './notebookWizard';
 
 const localize = nls.loadMessageBundle();
 
 export class NotebookWizardPage extends WizardPageBase<NotebookWizard> {
 	private inputComponents: InputComponents = {};
+	private inputComponentOriginalValues!: Map<string, string>;
```

Comment:
Also you're marking this as non-null but in the constructor you're only setting it in a certain case. So the non-null status is just wrong here. You shouldn't use that unless it's truly actually known to be safe to use without a null check - in this case I would suggest just using the null-coalescing operation instead when you call this below to get rid of the linting errors.

Reply:
removed inputComponentOriginalValues!